tests/std/tests/P0896R4_views_take_while/test.cpp

```diff
@@ -31,7 +31,7 @@ using pipeline_t = ranges::take_while_view<
 
 template <class Rng>
 concept CanViewTakeWhile = requires(Rng&& r) {
-    views::take_while(static_cast<Rng&&>(r), is_less_than<3>);
+    views::take_while(forward<Rng>(r), is_less_than<3>);

```

Comment:
why are we changing all these? I don't have a problem changing them but am curious.

Reply:
Just general cleanup. Stephan feels that `static_cast<meow&&>(woof)` harms readability enough to not be worth the slight increase in throughput over `forward<meow>(woof)`. We reached an accord that allows the cast form to be used in constraints, since the cost of specializing `forward` for argument types with which it may actually never be called is slightly more impactful than only instantiating when we know it will be called. Some of the allowed uses have leaked out into contexts where they aren't allowed, and I clean them up when I notice them.